lib/eofs/tests/test_coding_standards.py

```diff
@@ -1,33 +0,0 @@
-"""Test coding standards compliance."""
```

Comment:
@ajdawson I replaced this test with pre-commits. We can add more checks there if you want but I would do that in another PR to avoid bloating this one more than it already is.

Reply:
This is fine I think